features/add_menu_item.feature

```diff
@@ -0,0 +1,19 @@
+@javascript
+Feature: Visitor can add to order
+    As a visitor,
+    In order to select products I want to buy
+    I would like to be able to add products to an order
+
+    Background:
+        Given the following menu item exist
+            | name         | description                                                                             | price   | animal |
+            | Pupperoni    | Dried free-range, grass-fed beef sausage seasoned with sage, fennel and coriander       | 100     | dog    |
+        And I visit the site
+
+    Scenario: Visitor adds an appetizer to their cart
+        When I click on "Add to Order" on "Pupperoni"
+        Then I should see "Current Order:"
+        And I should see "Pupperoni"
+        And I should see "Total"
+        Then I should see '100'
```

Comment:
Please refactor to 
`And I should see '100'`

Reply:
Done